app/src/test/java/org/mozilla/fenix/gleanplumb/OnDiskMessageMetadataStorageTest.kt

```diff
@@ -43,7 +44,7 @@ class OnDiskMessageMetadataStorageTest {
 
     @Test
     fun `GIVEN metadata is not loaded from disk WHEN calling getMetadata THEN load it`() =
-        runBlockingTest {
+        runTest(UnconfinedTestDispatcher()) {
```

Comment:
Also curious about the usage of `UnconfinedTestDispatcher` here.

Reply:
Tested now. Here it doesn't seem to be needed. Doesn't break anything but is not required.
Will remove it in favor of the default `StandardTestDispatcher`.